crates/nargo_cli/tests/execution_success/generators/src/main.nr

```diff
@@ -0,0 +1,57 @@
+// TODO?
+// the syntax for these return types is very difficult to get right :/
+// for arguments this can be handled with a generic Env (or with Fn traits when we add them)
+// but for return types neither fo these will help, you need to type out the exact type
```

Comment:
Can you avoid TODOs in the code in the future? If it is an issue, we should track it on github issues so we do not forget about it